_docs/howto/redaxo_ddev.md

```diff
@@ -0,0 +1,87 @@
+---
+title: REDAXO mit DDEV Installieren
+authors: [crydotsnake]
+prio:
+---
+
+# REDAXO mit DDEV Installieren
+
+- [Was ist DDEV?](#was-ist-ddev)
+- [DDEV Voraussetzungen](#ddev-voraussetzungen)
+- [REDAXO Klonen](#redaxo-klonen)
+- [DDEV Konfigurieren](#ddev-konfigurieren)
+- [Projekt Starten](#projekt-starten)
+- [REDAXO Setup](#redaxo-setup)
+- [Geschafft](#geschafft)
+- [Tipps](#tipps)
+
+<a name="was-ist-ddev"></a>
+## Was ist DDEV ?
+
+DDEV ist ein auf Docker basierendes Open-Source-Tool. Es ermöglicht es, ganz schnell und einfach eine Lokale PHP Entwicklung aufzubauen. Standardmäßig gibt es in DDEV auch andere Vorkonfigurationen, für andere Systeme. Man kann aber auch einfach sein eigenes PHP Projekt mit DDEV Betreiben.
+
+<a name="ddev-voraussetzungen"></a>
+## DDEV Voraussetzungen
+
+- [Homebrew](https://brew.sh/index_de)
+- [Docker](https://docker.com)
+- [DDEV Anleitung](https://ddev.readthedocs.io/en/stable/)
+
+<a name="redaxo-klonen"></a>
+## REDAXO Klonen
+
+Als erstes, erstellt man sich ein Projekt Verzeichnis für das REDAXO Projekt. Dann klont man sich REDAXO in das Projekt Verzeichnis und führt im Terminal folgenden Befehl aus: `git clone https://github.com/redaxo/redaxo.git .`. Im Anschluss noch die submodule updaten `git submodule init && git submodule update`.
```

Comment:
die submodule haben wir ja mittlerweile net mehr.

Reply:
Okay, das wusste ich nicht.. Also kann ich die Zeile löschen?.

Reply:
@bloep Okay. Wo soll ich das am besten einfügen im Artikel?.

Reply:
Oder meinst du, denn REDAXO Installationspfad damit zu ersetzen?.

Reply:
habe mir grad nochmal angeschaut, was wir überhaupt in der root composer.json haben. Das ist für die meisten uninteressant und net notwendig zu installieren.
Daher würde ich einfach nur den Passus mit den submodules entfernen (wie es grad schon ist :wink: )

Reply:
Okay! :blush: !. Ich warte noch, ob @schuer vllt noch ein paar Verbesserungsvorschläge hat :).